### DISCUSSION TITLE
Would an extra pair of hands be useful?

### DISCUSSION THREAD

**Discussion by MikeWinter** [govuk-react/govuk-react#1200] [General]
I've worked with this library recently, encountering a few issues on the backlog ([`ErrorSummary` accessibility](https://github.com/govuk-react/govuk-react/issues/480) and [components conditionally revealed by form controls](https://github.com/govuk-react/govuk-react/issues/609)) and needing a basic [summary list](https://design-system.service.gov.uk/components/summary-list/).

I can't share these implementations directly (besides, they were not designed for broad use cases), but I'd be willing to help with these or other issues as time allows. I'm very grateful for the work already done here. If that'd be helpful, is there anywhere that'd be a good place to start?

**Comment by penx** [comment]
@MikeWinter thanks for offering to help. There’s certainly features I’d like to see added but I’m unsure of the best way to involve you and others in this. 

It’s worth including @rottitime in this discussion, as they have raised a couple of PRs recently that have been great!

I think ideally I’d like to have some kind of working group of users and those happy to contribute and maybe have a call every so often, maybe take it from there, but am open to other suggestions.

**Reply by rottitime** [reply]
@penx I am very open to a call every often. We use the govuk-react library almost every day for new projects. The library has been excellent and well developed but would love to see some new features to match:
https://design-system.service.gov.uk/components/

**Reply by MikeWinter** [reply]
> There’s certainly features I’d like to see added but I’m unsure of the best way to involve you and others in this.

That's understandable. It could certainly be difficult to simply throw folks in at the deep end, especially given some of the planned refactorings and API changes that contributors may not be aware of or fully understand.

> I think ideally I’d like to have some kind of working group of users and those happy to contribute and maybe have a call every so often, maybe take it from there, but am open to other suggestions.

Unfortunately, I don't think I have any insight when it comes to organising this kind of effort. My experience is in close-knit product teams, not distributed, open-source development. I'd like to be helpful if I can be, but that might not be possible — yet.

What might be useful for people that are interested in contributing is at least some guard rails. A sense of what sort of changes, suggestions or ideas would be productive at this stage, versus what might be harmful, distracting or otherwise burdensome. That might range from "Everything and anything" to "Designs for _X_" to "Issues with this label" to "Nothing except bug fixes until milestone _Y_".

**Comment by penx** [comment]
Looping in @jmanikiza , who has also offered help here https://github.com/govuk-react/govuk-react/discussions/1207

I would really like to hear from people using govuk-react to deliver production govuk projects so that we can set out a roadmap, and then we can open up contributions from others.

If you use govuk-react to deliver production govuk projects, it would be good to have a way to get in touch with you directly. Please add me somewhere I can DM you, such as [Twitter](https://twitter.com/penx) or [LinkedIn](https://www.linkedin.com/in/alasdairmcleay/), or let me know your email address (cc @rottitime @MikeWinter )
